verilog/formatting/token_annotator.cc

```diff
@@ -436,7 +437,13 @@ static WithReason<int> SpacesRequiredBetween(
     if (InRangeLikeContext(right_context)) {
       int spaces = right.OriginalLeadingSpaces().length();
       if (spaces > 1) {
-        spaces = 1;
+        // If ExcessSpaces returns 0 if there was a newline - prevents
```

Comment:
If we have changes here, we should also have a test in `token_annotator_test.cc` that covers both situations.

Reply:
There are already tests that look at the case of having no newlines. I have added a test for newline-containing spacing and it indeed passes, but only on the introduced changes, before it would fail